app/models/purchase.rb

```diff
@@ -1,12 +1,14 @@
 class Purchase < ActiveRecord::Base
-  validates :email, :item_id, presence: true
+  validates:item_id, :purchaser_id, presence: true
```

Comment:
There should either be a parenthesis or a white space between `validates:item_id`